README.md

```diff
@@ -19,6 +19,9 @@ Material Components for Tizen .NET help developers execute [Material Design](htt
 - **MCheckBox**
   A material checkbox allow the user to select one or more items from a set. ([guideline](https://material.io/design/components/selection-controls.html#checkboxes))
 
+- **MDialog**
```

Comment:
Add both `MAlertDialog` and `MConfirmationDialog` instead of `MDialog`

Reply:
Done.